plugins/cpp/parser/src/cppparser.cpp

```diff
@@ -358,6 +484,32 @@ bool CppParser::parse()
   return success;
 }
 
+void CppParser::initBuildActions()
+{
+  (util::OdbTransaction(_ctx.db))([&, this] {
```

Comment:
The `this` need not be captured, I think.

Reply:
Fixed in 64cb407581662e78675baf916a95dba2b765b3e4.